openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -139,6 +139,7 @@ runtime/containers/docker/TestCPUAwareness.java https://github.com/adoptium/aqa-
 runtime/containers/docker/TestCPUSets.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
 runtime/containers/docker/TestMemoryAwareness.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
 runtime/containers/docker/TestMisc.java https://github.com/adoptium/aqa-tests/issues/3629 linux-aarch64,linux-arm
+runtime/containers/docker/TestMemoryAwareness.java https://github.com/adoptium/aqa-tests/issues/3905 generic-all
```

Comment:
`runtime/containers/docker/TestMemoryAwareness.java` is already excluded in line 140. We should not add a new entry for it. The correct way is to update line 140 to `generic-all` with the new issue.
We may want to keep the old issue as a comment so we can still reference back.

Reply:
Thanks. I have merged three TestMemoryAwareness.java entry to one in ProblemList_openjdk8.txt.